PoseEuler pose
bool set_orientation
